common/errors_test.go

```diff
@@ -54,6 +54,7 @@ func TestDownstreamError_CreateDownstreamError_Timeout(t *testing.T) {
 	r.WriteHeader(http.StatusConflict)
 
 	resp := r.Result()
+	defer resp.Body.Close()
```

Comment:
This change has been fixed as part of separate pr - https://github.com/anz-bank/sysl-go/pull/253